src/test/java/com/faforever/client/vault/search/SearchControllerTest.java

```diff
@@ -125,7 +125,7 @@ public void testBuildQueryWithCheckboxAndQuery() throws Exception {
 
     when(specificationController.appendTo(any())).thenReturn(Optional.of(condition));
     when(condition.and()).thenReturn(qBuilder);
-    when(qBuilder.instant(eq("endTime"))).thenReturn(eq(instant));
+    when(qBuilder.instant(eq("endTime"))).thenReturn(instant);
```

Comment:
This line throws an Exception:
`Invalid use of argument matchers!
0 matchers expected, 1 recorded:
-> at com.faforever.client.vault.search.SearchControllerTest.testBuildQueryWithCheckboxAndQuery(SearchControllerTest.java:128)`

When using `qBuilder.instant("endTime")` as this Exception suggests, a NullPointerException is thrown (presumably because of the missing Matcher):
`java.lang.NullPointerException
	at com.github.rutledgepaulv.qbuilders.utilities.ObjectUtils.lambda$null$1(ObjectUtils.java:42)
	at java.base/java.util.stream.MatchOps$2MatchSink.accept(MatchOps.java:119)
	at java.base/java.util.stream.Streams$RangeIntSpliterator.tryAdvance(Streams.java:82)
	at java.base/java.util.stream.IntPipeline.forEachWithCancel(IntPipeline.java:163)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230)
	at java.base/java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.IntPipeline.allMatch(IntPipeline.java:518)
	at com.github.rutledgepaulv.qbuilders.utilities.ObjectUtils.lambda$init$2(ObjectUtils.java:42)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at com.github.rutledgepaulv.qbuilders.utilities.ObjectUtils.init(ObjectUtils.java:43)
	at com.github.rutledgepaulv.qbuilders.builders.QBuilder.prop(QBuilder.java:97)
	at com.github.rutledgepaulv.qbuilders.builders.QBuilder.instant(QBuilder.java:85)
	at com.faforever.client.vault.search.SearchControllerTest.testBuildQueryWithCheckboxAndQuery(SearchControllerTest.java:128)`